src/core/console.ts

```diff
@@ -1,24 +1,40 @@
-import filterConsole from 'filter-console'
+const consoleFilters = [
+  /^The above error occurred in the <.*?> component:/, // error boundary output
```

Comment:
I've slightly loosened what this will match, in case a `Wrapper` throws instead of the `TestComponent`